frontend/src/main/pages/LoginPage.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from "react"
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import CommonsList from "main/components/Commons/CommonsList";
+import { Container, Row, Col, Card, Button } from "react-bootstrap";
+import Background from './../../assets/HomePageBackground.jpg';
+
+import { useBackend } from "main/utils/useBackend";
+
+const LoginCard = () => {
+    return(
+        <Card style={{ width: '18rem' }}>
+        <Card.Body>
+            <Card.Title data-testid="loginPage-cardTitle">Welcome to Happier Cows!</Card.Title>
+            <Card.Text>
+            In order to start playing, please login.
+            </Card.Text>
+            <Button href="/oauth2/authorization/google" variant="primary">Log In</Button>
+        </Card.Body>
+        </Card>
+    )
+}
+
+export default function LoginPage() {
+    const { data: commons, error: commonError, status: commonStatus } = 
+    useBackend(
+      // Stryker disable next-line all : don't test internal caching of React Query
+      ["/api/commons/all"],
+      {  // Stryker disable next-line all : GET is the default, so changing this to "" doesn't introduce a bug
+        method: "GET",
+        url: "/api/commons/all"
+      },
+      []
+    );
```

Comment:
I have no idea why this call isn't ejecting a toast when the page renders, because we're getting HTTP 403s in the console.

Reply:
It isn't this call. It's the existence of <BasicLayout> inside the render() function.